<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddEditFollowUps</class>
 <widget class="QDialog" name="AddEditFollowUps">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Follow Up Info</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="3">
         <widget class="QComboBox" name="comboBox_Type">
          <item>
           <property name="text">
            <string>Return Consultation</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Prescription Renewal</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Referral</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Medical Test</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Details:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="3">
         <widget class="QLineEdit" name="lineEdit_Status">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Pending</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Status:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Type:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Due Date:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="3">
         <widget class="QPlainTextEdit" name="plainTextEdit_Details"/>
        </item>
        <item row="4" column="3">
         <widget class="QDateTimeEdit" name="dateTimeEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddEditFollowUps</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddEditFollowUps</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
